desktop/src/cli.rs

```diff
@@ -281,3 +281,125 @@ impl Opt {
         })
     }
 }
+
+// TODO The following enum exists in order to preserve
```

Comment:
Maybe we could just re-use the names used on the web for `KeyboardEvent` `code`. Or just go basic and use the integer key codes.

Reply:
I think the whole CLI needs to be reviewed before 1.0.0 and potentially changed in one PR. I believe our users would not appreciate it changing frequently. TBH I'd be aiming for preserving the behavior of CLI in this specific PR.

Reply:
We definitely should not change it in this PR. I was just trying to brainstorm ideas.

Reply:
I'd personally just use numeric codes in that case. Remapping buttons with CLI is obscure anyway and we'll have a nice GUI for that in the future :eyes: